test/graphql/mutations.tests.js

```diff
@@ -0,0 +1,516 @@
+          addPendingEvent(
+            txid: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            ownerAddress: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            name: "General test"
+            results: ["1", "2"]
+            numOfResults: 2
+            centralizedOracle: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            betStartTime: 1560965704
+            betEndTime: 1560965704
+            resultSetEndTime: 1560965704
+            language: "en-US"
+          ) {
+            ${MULTIPLE_RESULTS_EVENT}
+          }
+        }
+      `;
+      tester.test(false, valid, {});
+    });
+
+    it('Miss resultSetEndTime', async () => {
+      const valid = `
+        mutation {
+          addPendingEvent(
+            txid: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            ownerAddress: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            name: "General test"
+            results: ["1", "2"]
+            numOfResults: 2
+            centralizedOracle: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            betStartTime: 1560965704
+            betEndTime: 1560965704
+            resultSetStartTime: 1560965704
+            language: "en-US"
+          ) {
+            ${MULTIPLE_RESULTS_EVENT}
+          }
+        }
+      `;
+      tester.test(false, valid, {});
+    });
+
+    it('Miss language', async () => {
+      const valid = `
+        mutation {
+          addPendingEvent(
+            txid: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            ownerAddress: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            name: "General test"
+            results: ["1", "2"]
+            numOfResults: 2
+            centralizedOracle: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            betStartTime: 1560965704
+            betEndTime: 1560965704
+            resultSetStartTime: 1560965704
+            resultSetEndTime: 1560965704
+          ) {
+            ${MULTIPLE_RESULTS_EVENT}
+          }
+        }
+      `;
+      tester.test(false, valid, {});
+    });
+
+    it('Txid should be string', async () => {
+      const valid = `
+        mutation {
+          addPendingEvent(
+            txid: 12
+            ownerAddress: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            name: "General test"
+            results: ["1", "2"]
+            numOfResults: 2
+            centralizedOracle: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            betStartTime: 1560965704
+            betEndTime: 1560965704
+            resultSetStartTime: 1560965704
+            resultSetEndTime: 1560965704
+            language: "en-US"
+          ) {
+            ${MULTIPLE_RESULTS_EVENT}
+          }
+        }
+      `;
+      tester.test(false, valid, {});
+    });
+
+    it('ownerAddress should be string', async () => {
+      const valid = `
+        mutation {
+          addPendingEvent(
+            txid: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428"
+            ownerAddress: null
```

Comment:
add another case in here if `ownerAddress` is a number. when writing validation tests, think of all the different kinds of types that can potentially be passed in. even passing in an `object` or `array` can be a case,